TFT/src/User/API/UI/GUI.c

```diff
@@ -963,7 +962,7 @@ void _GUI_DispLabel(int16_t x, int16_t y, uint16_t index)
 {
   uint8_t tempstr[MAX_LANG_LABEL_LENGTH];
   if (loadLabelText((uint8_t*)&tempstr, index) == false) return;
-  _GUI_DispString(x, y, tempstr);
+    _GUI_DispString(x, y, tempstr);
```

Comment:
no indentation here (it's not for "if". The same for the following _GUI....

Reply:
don't know how they got indented. Maybe I was drunk.